packages/office-ui-fabric-react/src/components/HoverCard/docs/HoverCardOverview.md

```diff
@@ -3,4 +3,4 @@ HoverCards supplement content associated with a specific data element.
 Two scenarios to consider for your own use case from an accessibility point of view when deciding how the card should open and it's behavior after it opened:
 
 - Tabbing with a keyboard to the element triggering the HoverCard to open on focus. In this case no further navigation within the card is available and navigating to the next element will close the card. See Example 1.
-- Tabbing with a keyboard to the element triggering the HoverCard and wait to open till the hotKey is depressed. When card is opened it will automatically focus the first focusable element of the card content. See Example 2.
+- Tabbing with a keyboard to the element triggering the HoverCard and opening when the hotKey is depressed. When the card is opened it will automatically focus the first focusable element of the card content. See Example 2.
```

Comment:
Should it be `opening while` instead of `opening when`?

Reply:
I think it's weird to have `while` mixed with `depressed`. Maybe `pressed`, but `depressed` is the ending of an action, so it's like saying "while you aren't doing anything".

Reply:
@JasonGore could you please also correct a typo on line 3 `it's behavior` needs to be `its behavior`. Thanks for docs update.

Reply:
fixed the "its"

Reply:
It's it's when it is, otherwise it's its.